test/check-timezone

```diff
@@ -0,0 +1,206 @@
+        dt = DateAndTime(b, m)
+
+        i.open()
+        i.reach(i.steps.DATE_TIME)
+        dt.check_auto_date_time(dt.dbus_get_ntp_enabled())
+        dt.check_auto_timezone(True)
+        ntp_servers = dt.dbus_get_ntp_servers()
+        if ntp_servers:
+            dt.check_ntp_server(ntp_servers[0])
+        else:
+            print("WARNING: No NTP servers returned from backend, skipping NTP check.")
+        dt.check_timezone_label(dt.dbus_get_timezone())
+
+    def testManualDateTime(self):
+        """
+        Description:
+            Test manual entry of date and time.
+
+        Expected results:
+            - After disabling auto, manual values are applied to backend
+        """
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        lang = Language(b, m)
+        dt = DateAndTime(b, m)
+
+        i.open()
+        lang.select_locale("de_DE")
+        i.reach(i.steps.DATE_TIME)
+        dt.set_auto_date_time(False)
+        dt.set_date("2024-12-24")
+        dt.check_date("2024-12-24")
+        dt.set_time("18:30")
+        dt.check_time("18:30")
+        self.assertIn("2024-12-24T18:30", dt.dbus_get_system_datetime())
+
+    def testNtpServerSelection(self):
+        """
+        Description:
+            Test selecting another NTP server.
+
+        Expected results:
+            - Selected server is shown and set on backend
+        """
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        dt = DateAndTime(b, m)
+
+        i.open()
+        i.reach(i.steps.DATE_TIME)
+        dt.select_ntp_server("2.fedora.pool.ntp.org")
+        dt.check_ntp_server("2.fedora.pool.ntp.org")
+
+    def testManualTimezone(self):
+        """
+        Description:
+            Test manual timezone selection.
+
+        Expected results:
+            - Manual region/city changes backend timezone
+        """
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        dt = DateAndTime(b, m)
+
+        i.open()
+        i.reach(i.steps.DATE_TIME)
+        dt.set_auto_timezone(False)
+        dt.select_region("Europe")
+        dt.select_city("Prague")
+        dt.check_timezone_label("Europe/Prague")
+        self.assertEqual(dt.dbus_get_timezone(), "Europe/Prague")
+
+    def testTimeFormatSwitch(self):
+        """
+        Description:
+            Test switching time format.
+
+        Expected results:
+            - Time field display switches between 24h and AM/PM
+        """
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+        dt = DateAndTime(b, m)
+
+        i.open()
+        i.reach(i.steps.DATE_TIME)
+        dt.set_time_format("ampm")
+        dt.check_time_format("ampm")
+        dt.set_time_format("24")
+        dt.check_time_format("24")
+
+    def testTimezoneAndTimeformat(self):
+        """
+        Test full cycle of timezone, time format (24/ampm) and correct UTC backend saving.
```

Comment:
Keep the pattern for the docstrings always with 'Description.. \n Expected results:\n;'